priv/catalog/0B6DB2.yaml

```diff
@@ -48,7 +48,7 @@ remediation: |
     - https://documentation.suse.com/sle-ha/15-SP3/html/SLE-HA-all/cha-ha-storage-protect.html#pro-ha-storage-protect-sbd-config
     - https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/#id-adapting-the-sbd-configuration
 
-when: env.provider in ["azure", "nutanix", "kvm"]
+when: env.provider != "gcp" & env.provider != "aws"
```

Comment:
What about `env.provider !in ["gcp", "aws"]`? (notice the `!` before the `in`).
It looks more beautiful and scalable if we want to change the array

https://rhai.rs/book/language/in.html

Reply:
yes, it's simpler to enhance. Thanks